docs/support/contributing/style.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Style guide
+---
+
+# Documentation Style Guide
+
+This documentation aims to follow the [Google Developer Documentation Style Guide][google_style]. This style is not strictly enforced, but should be deferred to when options are being debated.
+
+
+## Sample code
```

Comment:
Feel like this should probably explain the principle of using styles that are friendly to novice developers, but I think some experienced developers will be surprised about conventions that diverge from best practices in most codebases (like formatting strings being vastly preferred to concatenation).

Reply:
Great callout, done.